modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -388,7 +388,9 @@ private void fixModelImports(Set<String> imports) {
                 composedSchemaSets.add(cm.oneOf);
                 for (Set<String> importSet : composedSchemaSets) {
                     for (String otherModelName : importSet) {
-                        cm.imports.add(otherModelName);
+                        if (!"none_type".equals(otherModelName)) {
```

Comment:
@spacether , this is one way to ensure the import below is not generated, but it's brittle. Is there a better way to skip these special case imports?

```python
try:
    from openapi_client.models import null
except ImportError:
    null = sys.modules[
        'openapi_client.models.null']
```